spec/services/target_matching_service_spec.rb

```diff
@@ -0,0 +1,63 @@
+require 'rails_helper'
+
+RSpec.describe TargetMatchingService do
```

Comment:
You can define something like 
`subject { described_class.new(target).give_me_matches }`

And then you can just call `subject` whenever you need the test to run that method.

Reply:
Thank you, that was helpful and made my code dried.